runtime/domain.c

```diff
@@ -901,7 +901,7 @@ void caml_handle_gc_interrupt() {
   if (Caml_state->requested_major_slice) {
     caml_ev_begin("dispatch_major_slice");
     Caml_state->requested_major_slice = 0;
-    caml_major_collection_slice(0);
+    caml_major_collection_slice(-1);
```

Comment:
Small comment, from what I gathered there was only two state possible, 0 and more, with a simple check.
Now that it can -1, 0 and more, maybe it is worth introducing a define for readability purpose?

Reply:
So I was using the -1, 0, other encoding that is used in stock 4.10.0:
https://github.com/ocaml/ocaml/blob/9d2a531ea9738f47d6fe71c6e0a0255dda613f4b/runtime/major_gc.c#L613-L619
I can add a define to make the -1, 0 easier to read.

Reply:
added some defines with comments to `major_gc.h`